examples/projections/nongeo/polar.py

```diff
@@ -55,57 +55,58 @@
 pygmt.config(FONT_TITLE="14p,Helvetica,black", FORMAT_GEO_MAP="+D")
```

Comment:
Since the titles contains parameters and arguments, I think a monospaced font is better.
```suggestion
pygmt.config(FONT_TITLE="14p,Courier,black", FORMAT_GEO_MAP="+D")
```